docs/asset-store/docs/03-04-asset-metadata-service.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Asset Metadata Service
+type: Details
+---
+
+The Asset Metadata Service is an HTTP server that exposes files metadata extraction functionality. It contains a simple HTTP endpoint which accepts `multipart/form-data` forms. The service extracts front matter YAML metadata from text files in arbitrary extension.
+
+The main purpose of the service is to provide metadata extraction for Asset Store. To use it, define `metadataWebhookService` in Asset and ClusterAsset custom resources.
```

Comment:
Please provide exact information how such webhook would look like

Reply:
I haven't written about it intentionally, because we don't know yet the final form. I guess it should be updated in task, where we add the functionality to Asset and ClusterAsset controllers.